content/influxdb/v2.0/reference/glossary.md

```diff
@@ -1092,13 +1092,16 @@ Related entries: [point](#point), [unix timestamp](#unix-timestamp), [RFC3339 ti
 ### token
 
 Tokens (or API tokens) verify user and organization permissions in InfluxDB.
-There are different types of athentication tokens:
+There are different types of authentication tokens:
 
-- **Operator token:** grants full read and write access to all resources in **all organizations in InfluxDB OSS 2.x**.
-  _InfluxDB Cloud does not support Operator tokens._
+{{% oss-only %}}
+
+- **Operator token:** grants full read and write access to all resources in **all organizations in InfluxDB OSS 2.x**. _InfluxDB Cloud does not support Operator tokens._
 - **All-Access token:** grants full read and write access to all resources in an organization.
 - **Read/Write token:** grants read or write access to specific resources in an organization.
 
+{{% /oss-only %}}
+
```

Comment:
Formatted as is, there won't be a list of token types in the cloud glossary. The suggestion below adds a cloud-specific list of token types.

```suggestion
{{% oss-only %}}

- **Operator token:** grants full read and write access to all resources in **all organizations in InfluxDB OSS 2.x**. _InfluxDB Cloud does not support Operator tokens._
- **All-Access token:** grants full read and write access to all resources in an organization.
- **Read/Write token:** grants read or write access to specific resources in an organization.

{{% /oss-only %}}
{{% cloud-only %}}

- **All-Access token:** grants full read and write access to all resources in an organization.
- **Read/Write token:** grants read or write access to specific resources in an organization.

{{% /cloud-only %}}

```